zhusuan/distributions/multivariate.py

```diff
@@ -403,3 +372,254 @@ def _log_prob(self, given):
 
     def _prob(self, given):
         return tf.exp(self._log_prob(given))
+
+
+class ExpConcrete(Distribution):
```

Comment:
Could this class inherit from Concrete? seems like except sample/log_prob, all others are the same